packages/docs/docs/communications/organizing-communications/organizing-communications.md

```diff
@@ -0,0 +1,167 @@
+# Organizing Communications
+
+## Background
+
+Communication in a patient portal can include many scenarios – patient to physician, physician to physician, device to physician, and more. This results in a large volume of communications within a portal, so ensuring that they are well organized is vital. For example, an organization may have staff on call to respond to patient messages. Without proper organization, messages may not reach the correct staff member.
+
+## Communication Organization Patterns
+
+There are various ways to organize communications using FHIR, but the most common are `Communication.category` and `Communication.topic`. Additionally, you can use FHIR attirbutes to organize communications using threading.
```

Comment:
typo: attributes

Reply:
Also, I don't believe attributes organize threading - am I missing something?  Did you mean `inResponseTo`?